student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+# student class
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] Kindly pass the `age` and `name` to the initialize method so you can pass them to the super class.